src/condor_schedd.V6/qmgmt.cpp

```diff
@@ -732,8 +733,21 @@ int GetSchedulerCapabilities(int /*mask*/, ClassAd & reply)
 	if (cmds && (cmds->size() > 0)) {
 		reply.Insert("ExtendedSubmitCommands", cmds->Copy());
 	}
-	dprintf(D_ALWAYS, "GetSchedulerCapabilities called, returning\n");
-	dPrintAd(D_ALWAYS, reply);
+	auto helpfile = scheduler.getExtendedSubmitHelpFile();
+	if ( ! helpfile.empty()) {
+		// if EXTENDED_SUBMIT_HELPFILE is not a URL, assume it is a small local file and return the content
+		if ((mask & 1) && ! IsUrl(helpfile.c_str())) {
```

Comment:
If we're using a mask rather than treating the `int` as a bool, shouldn't this by a symbolic (`#define`d) constant?

Reply:
done.